.github/workflows/plugins.yml

```diff
@@ -0,0 +1,115 @@
+name: plugins
+
+on:
+  # Automatically run CI on Release and Pre-Release tags and main branch
+  # (only if there are changes to relevant paths)
+  push:
+    tags:
+      - "xp-engine-plugin/v[0-9]+.[0-9]+.[0-9]+*"
```

Comment:
Nit: Can it just be `turing-plugin` for brevity in the tagging? Since it belongs to the `turing-experiments` repository, it would be clear enough that it's for XP, no? Also the job names: `turing-experiment-engine-xp-plugin-tests` -> `turing-plugin-tests`.